server/models/Review.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const reviewSchema = new mongoose.Schema({
+  user: {
+    firstName: String,
+    lastName: String,
+    profileImg: String,
+  },
+  rating: {
+    type: Number,
+    min: 1,
+    max: 5,
+    required: true,
+  },
+  message: String,
+  postDate: {
+    type: Date,
+    default: Date.now(),
+  },
```

Comment:
Mongoose can do this by passing a timestamp parameter to the Schema declaration

Reply:
changed to timestamps: true